cmd/caa-log-checker/main.go

```diff
@@ -114,6 +126,9 @@ func checkIssuances(scanner *bufio.Scanner, checkedMap map[string][]time.Time, t
 	if err := scanner.Err(); err != nil {
 		return err
 	}
+	if debug {
+		fmt.Fprintf(stderr, "lnum %d evaluated %d skipped %d\n", lNum, evaluatedCount, skipCount)
```

Comment:
Let's expand `lnum` to explain what it does. E.g. `lines read`.

Also as a future improvement it probably makes sense to plumb through a `logger` to this function, and log this at the debug level rather than writing it to `stderr`. That way we can log it unconditionally without interfering with code that reads stderr from this program. However I don't think that's necessary for this PR.